tests/test_dagger.py

```diff
@@ -25,62 +29,146 @@ def test_beta_schedule():
 
 
 def test_traj_collector(tmpdir):
-    env = gym.make(ENV_NAME)
+    venv = vec_env.DummyVecEnv([lambda: gym.make(ENV_NAME)])
```

Comment:
A `venv` with a single environment is a bit of a degenerate case. Maybe use the existing `venv` fixture, and merhaps even make the number of environments parameterized?

Reply:
Made both suggestions here